join-chat-to-teams-meeting/Chat Experience with File Sharing/client.js

```diff
@@ -0,0 +1,217 @@
+const callButton = document.getElementById("join-meeting-button");
+const hangUpButton = document.getElementById("hang-up-button");
+const callStateElement = document.getElementById('call-state');
+
+const messagesContainer = document.getElementById("messages-container");
+const chatBox = document.getElementById("chat-box");
+const sendMessageButton = document.getElementById("send-message");
+const messagebox = document.getElementById("message-box");
+
+var userId = '';
+var messages = '';
+var tokenString = '';
+
+async function init() {
+    const connectionString = "<SECRET_CONNECTION_STRING>";
+    const endpointUrl = "<ENDPOINT_URL>";
+
+    const identityClient = new CommunicationIdentityClient(connectionString);
+
+    let identityResponse = await identityClient.createUser();
+    userId = identityResponse.communicationUserId;
+    console.log(`\nCreated an identity with ID: ${identityResponse.communicationUserId}`);
+
+    let tokenResponse = await identityClient.getToken(identityResponse, [
+        "voip",
+        "chat",
+    ]);
+
+    const {
+        token,
+        expiresOn
+    } = tokenResponse;
+    tokenString = token;
+    console.log(`\nIssued an access token that expires at: ${expiresOn}`);
+    console.log(token);
+
+    const callClient = new CallClient();
+    const tokenCredential = new AzureCommunicationTokenCredential(token);
+    callAgent = await callClient.createCallAgent(tokenCredential);
+    callButton.disabled = false;
+
+    chatClient = new ChatClient(
+        endpointUrl,
+        new AzureCommunicationTokenCredential(token)
+    );
+
+    console.log('Azure Communication Chat client created!');
+}
+
+init();
+
+callButton.addEventListener("click", async () => {
+    // join with meeting link
+    call = callAgent.join({
+        meetingLink: meetingLinkInput.value
+    }, {});
+
+    call.on('stateChanged', () => {
+        callStateElement.innerText = call.state;
+    })
+    // toggle button and chat box states
+    chatBox.style.display = "block";
+    hangUpButton.disabled = false;
+    callButton.disabled = true;
+
+    messagesContainer.innerHTML = messages;
+
+    console.log(call);
+
+    // open notifications channel
+    await chatClient.startRealtimeNotifications();
+
+    // subscribe to new message notifications
+    chatClient.on("chatMessageReceived", (e) => {
+        console.log("Notification chatMessageReceived!");
+
+        // check whether the notification is intended for the current thread
+        if (threadIdInput.value != e.threadId) {
+            return;
+        }
+
+        if (e.sender.communicationUserId != userId) {
+            renderReceivedMessage(e);
+        } else {
+            renderSentMessage(e.message);
+        }
+    });
+
+    chatThreadClient = await chatClient.getChatThreadClient(threadIdInput.value);
+});
+
+async function renderReceivedMessage(event) {
+    messages += '<div class="container lighter">' + event.message + '</div>';
```

Comment:
minor: (applies to all), maybe use string template literals for more readability? (except for the multi-line strings)
```
`<div class="container lighter"> ${event.message} </div>`;
```

Reply:
good call updated!